scope.go

```diff
@@ -379,6 +405,31 @@ func (scope *Scope) ApplyToEvent(event *Event, hint *EventHint) *Event {
 		}
 	}
 
+	// Apply the trace context to errors if there is a Span on the scope. If
+	// there isn't then fall back to the propagation context.
+	if event.Type != transactionType {
```

Comment:
@cleptric There was a comment on old type to use this for transactions as well?

Reply:
We have to throw this on all events, so check.ins, errors, transactios, etc.